Makefile

```diff
@@ -122,8 +119,15 @@ MMCTL_REL_TO_DOWNLOAD = $(shell scripts/get_latest_release.sh 'mattermost/mmctl'
 
 all: run ## Alias for 'run'.
 
+-include config.override.mk
+include config.mk
 include build/*.mk
 
+RUN_IN_BACKGROUND ?=
```

Comment:
`config.mk` has already defined `RUN_IN_BACKGROUND` as `true` -- do we still need to re-declare it here?

Reply:
Actually, is subtle, but `RUN_IN_BACKGROND` is defining `&` not `true` or `false`, I use the `RUN_SERVER_IN_BACKGROUND` to initialize that, to avoid some code repetition later.

Reply:
Ah, yes, that makes sense. Thank you for clarifying :)